tests/run_tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+cd /tests
+
+# Validate hipcc works
+hipcc square.cpp -o square.out --amdgpu-target=gfx801
+ret=$?
+if [ $ret -ne 0 ]; then
+    exit 1
+fi
+
+# Get DNNMark loaded
+git clone https://github.com/doody1986/DNNMark.git
+cd DNNMark/
+git apply ../dnnmark.patch
+./setup.sh HIP
+cd build
+make
+
+# Get the file needed for the DNNMark program
```

Comment:
Please add a comment to make it clear why we have 8cc7... here ... it is because that's the path in 1.2.0 for the specific benchmark you're running as the test.

Separate: as we add more tests, it would require a ton of paths to be listed here which is very unwieldy.  Nothing to fix now, but we should discuss later.

Reply:
My immediate thought for more tests would be have another file with the 8cc7... in them, and then loop through that file. Although that may not work as easily for generating the files if they're not all e.g. MIOpenSoftmax

Reply:
That may work, although in practice there could be thousands of such SHAs.